homework/src/index.js

```diff
@@ -30,18 +33,70 @@
     return elem;
   }
 
+  function renderRepository(container, repository) {
+    container.innerHTML = '';
+    //const root = document.getElementById('root');
+    const table = createAndAppend('table', container);
+    const tBody = createAndAppend('tBody', table);
+    const trRepository = createAndAppend('tr', tBody);
+    trRepository.className = "table-info";
+    createAndAppend('td', trRepository, {html: "Repository:"});
+    //createAndAppend('td', trRepository, {html: repository.name});
+    const link = createAndAppend('a', trRepository,{html: repository.name});
+    link.setAttribute('href', repository.html_url);
+    const trDescripton = createAndAppend('tr', tBody);
```

Comment:
There is a spelling error here flagged by VSCode: `trDescripton` -> `trDescription`.

Reply:
Hi Jim, I firstly would like to thank for your feedback. I will try to correct missing parts based on your comments. I actually spent a significant amount of effort and time to create this homework. I am aware that only minimal requirements implemented,  and it is not because I tried the complete it at last minute. It was the best I could do. 
       I've just finished the contributors part and I am still working to make it responsive. After that, I will do "promises". 

Thanks..